app/src/main/java/org/isoron/uhabits/io/HabitsCSVExporter.java

```diff
@@ -60,6 +60,20 @@ private void writeHabits() throws IOException
         FileWriter out = new FileWriter(exportDirName + filename);
         generateFilenames.add(filename);
         Habit.writeCSV(habits, out);
+
+        filename = "AllCheckmarks.csv";
+        new File(exportDirName).mkdirs();
+        out = new FileWriter(exportDirName + filename);
+        generateFilenames.add(filename);
+        CheckmarkList check = new CheckmarkList();
+        check.writeCSVMultipleHabits (habits, out);
+
+        filename = "AllScores.csv";
+        new File(exportDirName).mkdirs();
+        out = new FileWriter(exportDirName + filename);
+        generateFilenames.add(filename);
+        ScoreList score = new ScoreList();
```

Comment:
Please, don't instantiate ScoreList and CheckmarkList here. Make the methods `writeCSVMultipleHabits` static instead, so that you can call `Score.writeCSVMultipleHabits`. Also, I would rename the method to simply `writeCSV`. From the arguments, it is clear that it works with multiple habits.